jail-ftp.yml

```diff
@@ -23,7 +23,8 @@
   - role: user
     user_id: buildbot
     user_name: Buildbot Uploader
-    authorized_key: >
+    authorized_key: |
       ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC7YoNGKRZJxVbvxQpGBX1FuhWC0Nq2+fe+wVD16cgaOk4iDjeKdQ4eZACIlcMi5REXfTO+e9Hzmr85DUmy+BAKx0PqzLj6BhmnZnoQbQe3WAsmWSIuUSqKqilM4RnkDUWrDvhdyMGIH4ELfrebJt5ZnNLbOTk6BAQiygF/1xNeYPvcjxfUG1NXUHlKJ9qnf1cph7Pw6YOe8SF8kJTxwDiSJSG0aGsfas4zLVgmcPHdjsFRuOFOxFlVpFKzu3toaZyXPSeTOAaoaIHkfaEahBExXyd7b8L/0fcFumWK6qwIA5dFDWGkIaEMBdX6/giV+gvz9UJQ75gxVtn0y8MbFEMD9iduGbIX5FSOFNmbEkNubW5M74G7igMtqk+dUbVSMOoy4NO2vPnXH0iBMPkkzPEmAro3/q/J5QTEbIB39H99dbGnOR1iwbBhCqVJERAAECteoHsXKHsNhL9IYqDp2KKPVHmxZybqtJo4SKSzNvjfK50ojYcIAAVlBsFZ+9p0Aj8CADA3L0Ont7RPs9Dcoad1la9kkC1v6Bol5O1huRM8qjXRPOT1W01JH+UvM8r1EKNKRgV/uVr+Zl1Ln84GvPMX4cdz+ky2L5mlE9w6RonBqJnJg16i7eQXCE6iwOjQ2XHK+TBeoQDGHmww+ACUmVx9NR+I4Q9cWMjteLx3Q+YMtQ== buildbot@ftp.buildbot.net
```

Comment:
I think this key is for the old buildbot.net buildslave.  So it should probably be removed.